apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -540,20 +584,15 @@ export default class ExportCKLModal extends Vue {
     return results;
   }
 
-  validateIpAddress(value: string): boolean | string {
-    if (!value) {
-      return true;
-    }
-    const ipPattern = /^(\d{1,3}\.){3}\d{1,3}$/;
-    return ipPattern.test(value) || 'Invalid IP Address Format';
-  }
-
-  validateMacAddress(value: string): boolean | string {
-    if (!value) {
-      return true;
+  validateFormat(field: typeof ValidationProperties, name: string): string[] {
+    const errors: string[] = [];
+    const dirty = _.get(field, '$dirty');
+    const invalid = _.get(field, '$invalid');
+    if (!dirty) {
+      return [];
     }
-    const macPattern = /^([0-9A-Fa-f]{2}[:-]){5}[0-9A-Fa-f]{2}$/;
-    return macPattern.test(value) || 'Invalid MAC Address Format';
+    invalid && errors.push(`Invalid ${name} Format`);
```

Comment:
While this works, javascript is a language that's mostly built on the concept of statements, not expressions that do side effects like this.

I would rewrite this into the more typical if statement structure (`if (invalid) { errors.push }`) or make the return statement a ternary (`return invalid ? errors + ['invalid format'] : errors`).

Reply:
This has been refactored away. No longer an issue